openedx/core/djangoapps/content/learning_sequences/api/permissions.py

```diff
@@ -31,20 +27,5 @@ def can_see_all_content(requesting_user: types.User, course_key: CourseKey) -> b
 
     There's no need to run processors to restrict results for these users.
     """
-    return (
-        GlobalStaff().has_user(requesting_user) or
-        CourseStaffRole(course_key).has_user(requesting_user) or
-        CourseInstructorRole(course_key).has_user(requesting_user)
-    )
-
-
-def can_see_content_as_other_users(requesting_user: types.User, course_key: CourseKey) -> bool:
```

Comment:
This method was removed in favor of a direct call to `has_access` in the view before masquerading. This was (A) to keep consistent with every other call to `setup_masquerade` where they call `has_access` directly and (B) the abstraction offered here (e.g. where we might add researcher roles in the future) felt weird to only offer inside LS - if we wanted that abstraction, ideally this method would exist in the masquerading module. But until that's done, I figured I'd just keep doing what other masquerade code is doing.